### ISSUE TITLE
Pytorch YoloV5 model with GradCAM

### ISSUE THREAD

**Issue by ugroznc** [jacobgil/pytorch-grad-cam#463]
I have trained an xlarge YoloV5 model using DIOR Dataset with Ultralytics. I can not use it with GradCAM. Is there any way that I can use my model with GradCAM?

Besr Regards.

**Comment by jS5t3r** [comment]
U can take any layer of the model. Usually, the last block is used for best results.
It can be tricky and you might need to add an Identity layer to the pre-trained model, which is useful to get the gradients... applying norm1 or norm2 would help, too.
